packages/loader/container-loader/src/container.ts

```diff
@@ -1453,6 +1460,25 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private processRemoteMessage(message: ISequencedDocumentMessage): IProcessMessageResult {
         const local = this._clientId === message.clientId;
 
+         // Report if we're getting messages from a clientId that is not in the quorum
+         // or we previously marked inactive
+         if (!local) {
+            const client = this._protocolHandler?.quorum.getMember(message.clientId);
+            const missingFromQuorum = client === undefined;
```

Comment:
This should never happen. By definition, the client should always be in the quorum.